pkg/config/ods.go

```diff
@@ -139,11 +139,17 @@ func (e Environment) Validate() error {
 	if len(e.Name) == 0 {
 		return errors.New("name of environment must not be blank")
 	}
-	pattern := "^[a-z-]*$"
+	pattern := "^[a-z][a-z0-9-]*[a-z]$"
```

Comment:
I am not quite sure why we came up with a pattern that disallows numbers at the end. From the links for the naming rules at [Namespaces and DNS ](https://kubernetes.io/docs/concepts/overview/working-with-objects/namespaces/#namespaces-and-dns) and in [Names](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-label-names) it looks like alphanumeric would be fine.

Reply:
Unfortunately I don't, but I do remember that we wanted to avoid numbers at the beginning and at the end. Maybe we just enforce this to be on the safe side in case there was a real reason we don't know anymore?